src/cli/test_command.zig

```diff
@@ -972,8 +972,9 @@ export fn BunTest__shouldGenerateCodeCoverage(test_name_str: bun.String) callcon
         break :brk zig_slice.slice();
     };
 
-    // always ignore node_modules.
-    if (bun.strings.contains(slice, "/node_modules/") or bun.strings.contains(slice, "\\node_modules\\")) {
+    // always ignore node_modules unless include_node_modules is true
+    if (!bun.JSC.Jest.runner.?.test_options.coverage.include_node_modules and 
+        (bun.strings.contains(slice, "/node_modules/") or bun.strings.contains(slice, "\\node_modules\\"))) {
```

Comment:
Bun CI is failing to build this https://buildkite.com/bun/bun/builds/17101:
```zig
src/cli/test_command.zig:976:22: error: root source file struct 'bun.js.test.jest' has no member named 'runner'
    if (!bun.JSC.Jest.runner.?.test_options.coverage.include_node_modules and
         ~~~~~~~~~~~~^~~~~~~
src/bun.js/test/jest.zig:1:1: note: struct declared here
const std = @import("std");
^~~~~
referenced by:
    BunTest__shouldGenerateCodeCoverage: src/cli/test_command.zig:964:1
```

Reply:
Fixed. removed JSC